lib/membrane_flac_plugin/membrane_flac_parser.ex

```diff
@@ -1,13 +1,13 @@
-defmodule Membrane.Element.FLACParser do
+defmodule Membrane.FLACParser do
```

Comment:
the convention is rather to have `FLAC.Parser`

Reply:
Yeah but we already have `Membrane.FLACParser.Parser` so if we changed this to `Membrane.FLAC.Parser` we would have `Membrane.FLAC.Parser.Parser`. Maybe `Membrane.FLACParser.Parser` should be renamed to `Membrane.FLAC.Parser.Native`? I am not sure :thinking:

Reply:
That's not really native, we used to call such stuff `Parser.Helper`. It could also be `Parser.Engine` like in SFU